source/jquery.flot.js

```diff
@@ -2508,30 +2533,35 @@ Licensed under the MIT license.
                     var found = findNearbyPoint(s, mouseX, mouseY, radius, smallestDistance, computeDistance);
                     if (found) {
                         smallestDistance = found.distance;
-                        item = [i, found.dataIndex];
+                        items.push([i, found.dataIndex]);
                     }
                 }
 
-                if (s.bars.show && !item) { // no other point can be nearby
+                if (s.bars.show && !items.length) { // no other point can be nearby
```

Comment:
If we're returning multiple nearby items now, why not also return the nearest bar plot point?

Reply:
Fixed.